doc/nrf/known_issues.rst

```diff
@@ -1702,7 +1740,22 @@ NCSDK-7982: Partition manager: Incorrect partition size linkage from name confli
 DFU and FOTA
 ============
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0
+.. rst-class:: v2-1-2 v2-1-1 v2-1-0
+
+NCSDK-18357: Serial recovery does not work on nRF5340 network core
```

Comment:
Hi @nordicjm,
Cherry-pick of `27be2a94da940558c74878611b44506b2b57f7ec` to sdk-nrf could be a workaround here, right?

Reply:
For what release? These are fixed in 2.2.0 and will be fixed in 2.1.3 - see https://github.com/nrfconnect/sdk-nrf/pull/9600 and https://github.com/nrfconnect/sdk-nrf/pull/9601 for 2.1.3 backports

Reply:
A note for this and the one below can be added that `sdk-mcuboot` repo can have the following commits cherry-picked to resolve them (note that they have to be in this order): `11c5149a7dd6c9f7581e971656bae461c135e31a`, `db3da7f4f3eefffb00d1900c231b84386eb5440e`, `78fd7ff9f31d88b9119734965e2afdda0cb9689a`

Reply:
Sorry, please remove these commits and leave both of these blank, they do not apply cleanly